test/markdownlint-test-repos.js

```diff
@@ -171,6 +171,7 @@ test("https://github.com/mkdocs/mkdocs", (t) => {
   const rootDir = "./test-repos/mkdocs-mkdocs";
   const globPatterns = [
     join(rootDir, "README.md"),
+    join(rootDir, "CHANGELOG.md"),
```

Comment:
This file is not present in the mkdocs repo.

Reply:
Removed.